modules/virt-attaching-virtio-disk-to-windows.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/creating_vms/virt-installing-qemu-guest-agent.adoc
+
+:_content-type: PROCEDURE
+[id="virt-attaching-virtio-disk-to-windows_{context}"]
+= Attaching VirtIO container disk to Windows VMs during installation
+
+You must attach the VirtIO container disk to the Windows VM to install the necessary Windows drivers. This can be done during creation of the VM or to an existing VM.
+
+.Procedure
+
+. When creating a Windows VM xref:../creating_vms_rh/virt-creating-vms-from-templates.html#virt-creating-vm-from-template_virt-creating-vms-from-templates[from a template], click *Customize VirtualMachine*.
+. Select the checkbox to enable the *Mount Windows drivers disk* option.
+. Click the *Customize VirtualMachine parameters* button.
+. Click *Create VirtualMachine*. During the creation, the `virtio-win` SATA CD disk will be attached to the VM.
+
+:_content-type: PROCEDURE
+[id="virt-attaching-virtio-disk-to-windows-existing_{context}"]
+= Attaching VirtIO container disk to an existing Windows VM
```

Comment:
You cannot have 2 level-1 headers in the same Asciidoc file. That's probably why your preview build is not working. This should be a separate module.